app/controllers/apps_controller.rb

```diff
@@ -17,7 +21,7 @@ def create
 
     if @app.save
       params[:pet_ids].each do |id|
-        PetApp.create(pet_id: id, app: @app)
+        PetApp.create(pet_id: id, app_id: @app.id, approved: false)
```

Comment:
not sure how we were getting away with this old code

Reply:
Not sure either but its working...for the love of god its working